File: Day_1/Q3.java
import java.util.*;
class Q3 {
    //Calculating The Factorial 
    static int factorial(int n){
        int fact = 1;
        for(int i=1;i<=n;i++){
            fact*=i;
        }
        return fact;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        // Printing The Result
        if(n < 0) System.out.println("Invalid Input"); // Checking For Valid Input
        else System.out.println("The Factorial Of " + n + " Is " + factorial(n));
        sc.close();
    }
}

File: Day_1/Q3.cpp
#include <iostream>
using namespace std;
// Calculating The Factorial
int factorial(int n)
{
    int fact = 1;
    for (int i = 1; i <= n; i++)
    {
        fact *= i;
    }
    return fact;
}
int main()
{
    int n;
    cin >> n;
    // Printing The Result
    if(n < 0) //Checking For Valid Input
    {
        cout << "Invalid Input";
        return 0;
    } 
    cout << "The Factorial Of " << n << " Is " << factorial(n);
    return 0;
}